docs/developer-docs/setup/cycles/cycles-ledger.md

```diff
@@ -0,0 +1,158 @@
+# Cycles ledger
+
+## Overview
+
+When developing and deploying canisters on the Internet Computer, developers need to have **cycles** to pay for the resources used by their canisters. The **cycles ledger** canister has been developed as an alternative to the **cycles wallet** canister for cycles management. 
+
+The biggest difference between the cycles ledger and cycles wallet is that the cycles ledger is a single global ledger canister, whereas the cycles wallet canister must be deployed locally to be used. Additionally, the cycles wallet has some downsides that the cycles ledger remedies. Some of these are:
+
+- Cycles wallets are a complex solution to associate cycles with principal IDs.
+- Cycles wallets consume cycles themselves, meaning developers need to pay cycles to store cycles.
+- Since the cycles wallet canister must be deployed locally, if a cycles wallet canister ID is lost, the cycles in that cycles wallet canister are also lost. 
+
+The cycles ledger simplifies cycle management by providing the ability for principal IDs to hold cycles. 
+
+The cycles ledger complies with the ICRC-2 standard. In addition to standard ledger functionality, the cycles ledger canister also interacts with the NNS canisters and user canisters to provide cycles ledger-specific functions, such as:
+
+- Accepting incoming cycles sent from other canisters.
+- Sending cycles to other canisters.
+- Creating new canisters using cycles.
+
+:::caution
+It is important to note that the cycles ledger does not provide the ability to make arbitrary calls to other canisters with cycles, while the cycles wallet can. This is because open call contexts may cause complications with the cycles ledger. 
+
+If you need to make arbitrary calls to other canisters with cycles, a cycles wallet can be used as before. The cycles ledger can be used to load the cycles wallet with cycles.
+:::
+
+## Obtaining ICP
+
+To use the cycles ledger, first you need ICP tokens. There are a few different ways to obtain ICP tokens, such as:
+
+- Purchasing ICP tokens directly through an exchange that lists ICP tokens available for trade. To see all available places to purchase ICP tokens, check out this [page](https://coinmarketcap.com/currencies/internet-computer/markets/).
+- Trading other tokens for ICP on a decentralized exchange.
+- Receiving tokens as rewards for participating in the governance of the Internet Computer.
+- Receiving a grant of tokens through the Internet Computer Association (ICA) or the DFINITY Foundation.
+- Receiving tokens as remuneration for providing computing capacity as a node provider.
+
+To obtain ICP, you also need to have an account to transfer those tokens into. To get your account ID, first determine which dfx identity you are using with the command:
```

Comment:
This sentence is not very clear. If I buy tokens on an exchange, I already have ICP, so I don't need to use `dfx` or somehow set up an account somewhere.

Maybe it should be a different section? Now that you have ICP, the next step is to transfer ICP to an account controlled via `dfx`, right?

Reply:
Revised to "Once you have obtained ICP, you also need to have an account controlled via `dfx` to transfer those tokens into. This step is necessary to create cycles and canisters. To get an account ID for an account controlled via `dfx`, first determine which `dfx` identity you are using with the command:"